flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/main/java/org/apache/flink/cdc/connectors/paimon/sink/v2/PreCommitOperator.java

```diff
@@ -110,17 +110,15 @@ public void snapshotState(StateSnapshotContext context) throws Exception {
         super.snapshotState(context);
         long checkpointId = context.getCheckpointId();
         if (!multiTableCommittables.isEmpty()) {
-            multiTableCommittables.forEach(
-                    (multiTableCommittable) ->
-                            LOGGER.debug(
-                                    "Try to commit for {}.{} : {} in checkpoint {}",
-                                    multiTableCommittable.getDatabase(),
-                                    multiTableCommittable.getTable(),
-                                    multiTableCommittables,
-                                    checkpointId));
             WrappedManifestCommittable wrappedManifestCommittable =
                     storeMultiCommitter.combine(checkpointId, checkpointId, multiTableCommittables);
+            long commitStart = System.currentTimeMillis();
             storeMultiCommitter.commit(Collections.singletonList(wrappedManifestCommittable));
+            LOGGER.info(
+                    "Commit for {} in checkpoint {} takes {} ms",
+                    wrappedManifestCommittable,
```

Comment:
In some cases where users have reported data loss or anomalies, we need to rely on commit information for troubleshooting. 
However, if we adjust the log level of the job later, it may not reflect the previous situation. 
Therefore, I suggest changing it to the info level.